packages/gatsby-transformer-sharp/src/extend-node-type.js

```diff
@@ -154,10 +154,22 @@ const fixedNodeType = ({
         type: ImageCropFocusType,
         defaultValue: sharp.strategy.attention,
       },
+      fit: {
+        type: ImageFitType,
+        defaultValue: sharp.fit.cover,
+      },
+      background: {
+        type: GraphQLString,
+        defaultValue: `rgba(0,0,0,1)`,
+      },
       rotate: {
         type: GraphQLInt,
         defaultValue: 0,
       },
+      trim: {
+        type: GraphQLInt,
```

Comment:
After another look, it seems that the type should be `GraphQLFloat` per [libvps](https://github.com/libvips/libvips/blob/master/libvips/arithmetic/find_trim.c#L56)